gatsby-browser.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import Layout from './src/components/Layout';
+
+export function wrapPageElement({ element, props }) {
+  // eslint-disable-next-line react/jsx-filename-extension
+  return <Layout {...props}>{element}</Layout>;
+}
```

Comment:
If we are using TS maybe we should also do it in configuration files?

Have you tried this one https://www.gatsbyjs.com/plugins/gatsby-plugin-ts-config/?